ql/src/java/org/apache/hadoop/hive/ql/exec/repl/util/ReplUtils.java

```diff
@@ -187,4 +192,12 @@ public static PathFilter getEventsDirectoryFilter(final FileSystem fs) {
       }
     };
   }
+
+  public static boolean isFirstIncDone(Map<String, String> parameter) {
+    if (parameter == null) {
+      return true;
+    }
+    String compFlag = parameter.get(ReplUtils.REPL_FIRST_INC_PENDING_FLAG);
```

Comment:
compFlag doesn't look relevant. Can change it.

Reply:
done